homeassistant/components/vesync/fan.py

```diff
@@ -110,9 +110,9 @@ def speed_count(self) -> int:
         )
 
     @property
-    def preset_modes(self):
+    def preset_modes(self) -> list[str]:
         """Get the list of available preset modes."""
-        return PRESET_MODES[SKU_TO_BASE_DEVICE.get(self.device.device_type)]
+        return PRESET_MODES[SKU_TO_BASE_DEVICE[self.device.device_type]]
```

Comment:
Agree it would already have caused issues. The current change will result in the same thing if it was an issue already. (So it didn't get more broken).